src/main/scala/io/viash/config/Config.scala

```diff
@@ -35,6 +35,8 @@ import io.viash.config_mods.ConfigMods
 import java.nio.file.Paths
 
 import io.viash.schemas._
+import java.io.ByteArrayOutputStream
+import cats.instances.boolean
```

Comment:
It seems to me that this `import cats.instances.boolean` is an accidental import

Reply:
true!